cpp/daal/src/algorithms/k_nearest_neighbors/kdtree_knn_classification_train_dense_default_impl.i

```diff
@@ -39,7 +39,7 @@
 #include "src/algorithms/k_nearest_neighbors/kdtree_knn_impl.i"
 #include "src/algorithms/engines/engine_batch_impl.h"
 
-#if defined(__INTEL_COMPILER_BUILD_DATE)
```

Comment:
How does `__INTEL_COMPILER_BUILD_DATE` map to `DAAL_INTEL_CPP_COMPILER`? Was it previously simply a synonym to `__INTEL_COMPILER`? Or is there more to pay attention to?

Reply:
yes.